docs/integrations/bridges/cbridge.md

```diff
@@ -61,6 +61,16 @@ After the transaction, you can see
 
 and check out your Metamask!
 
+## Understanding the difference betweeen USDT vs. cUSDT
+
+Recently, Tether announced the launch of USDT on Kusama (a scalable network of specialized blockchains built using Substrate and nearly the same codebase as Polkadot), which is the native USDT token on the Kusama blockchain.
+
+Shiden Network is a multi-chain decentralized application layer on Kusama Network. cUSDC on Shiden is a "bridged form of Ethereum USDT" that was bridged from the cBridge. <strong>Because of this, cUSDT is not the same as native USDT.</strong> Tether products do not support cUSDT. You cannot send cUSDT to a Tether deposit address. It is important to note that <strong>you cannot use cUSDT in any WASM environemnt but only in EVM environement.</strong> You can use USDT which is a native USDT token on the Kusama blockchain on both WASM and EVM enviroment.
```

Comment:
Celer bridge is only active on Astar so they will not bridge USDT to Shiden. MultiChain is our partner on Shiden. 
What do you mean by Tether products?

WASM projects built on Astar will be able to use the USDT registered as XC20 (which is the native USDT minted on Statemint). By register USDT as XC20 we also will have USDT (native) available as smart contract that you can find in our XCM asset list in the docs.

Reply:
I updated the sentence.